src/main/java/com/adyen/service/LegalEntityManagement/BusinessLineService.java

```diff
@@ -0,0 +1,39 @@
+package com.adyen.service.LegalEntityManagement;
+
+import com.adyen.Client;
+import com.adyen.Service;
+import com.adyen.constants.ApiConstants;
+import com.adyen.model.legalEntityManagement.BusinessLine;
+import com.adyen.model.legalEntityManagement.BusinessLineInfo;
+import com.adyen.model.legalEntityManagement.JSON;
+import com.adyen.service.exception.ApiException;
+import com.adyen.service.resource.LegalEntityManagementResource;
+
+import java.io.IOException;
+
+public class BusinessLineService extends Service {
+    public BusinessLineService(Client client) {
+        super(client);
+        new JSON();
+    }
+
+    public BusinessLine create(BusinessLineInfo request) throws IOException, ApiException {
+        String jsonRequest = request.toJson();
+        LegalEntityManagementResource resource = new LegalEntityManagementResource(this, "/businessLines");
+        String jsonResult = resource.request(jsonRequest);
+        return BusinessLine.fromJson(jsonResult);
+    }
+
+    public BusinessLine retrieve(String businessLineId) throws IOException, ApiException {
+        LegalEntityManagementResource resource = new LegalEntityManagementResource(this, String.format("/businessLines/%s", businessLineId));
+        String jsonResult = resource.request(null, ApiConstants.HttpMethod.GET);
+        return BusinessLine.fromJson(jsonResult);
+    }
+
+    public BusinessLine update(String businessLineId, BusinessLineInfo request) throws IOException, ApiException {
+        String jsonRequest = request.toJson();
```

Comment:
The request body `jsonRequest` is not sent (unused variable).

Reply:
good catch :)